aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -124,7 +125,6 @@ void rsqrt_kernel_cuda(TensorIteratorBase& iter) {
       iter.common_dtype(), "rsqrt_cuda",
       [&]() {
         gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
-          // In CUDA, ::rsqrt is overloaded for float and at::Half here is implicitly cast to float.
```

Comment:
I think we want to keep it here as we actually pass `at::Half` via the dispatch.